packages/tideline/plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -60,8 +60,12 @@ class BasicsChartNoSize extends React.Component {
     this.log.debug("Mounting...");
 
     const { tidelineData, bgClasses, bgUnits, patient, permsOfLoggedInUser } = this.props;
-    const basicsData = _.cloneDeep(tidelineData.basicsData);
 
+    if (!tidelineData.basicsData) {
+      return;
```

Comment:
It's just a guading if to avoid an app crash and display a "no data" message.
This in production with real user, should never be the case, but can be when we use our data-loader, with almost empty patient data.